mlrun/db/filedb.py

```diff
@@ -154,6 +154,11 @@ def list_runs(
             raise mlrun.errors.MLRunInvalidArgumentError(
                 "Runs partitioning not supported"
             )
+        if uid and isinstance(uid, list):
```

Comment:
I'd check Iterable and not just list, in case provided by tuple.
also, u allow it via function signature but protect it from being a list, why?

Reply:
`filedb` is a legacy thingie that we don't really support anymore (well, it's still there, but we don't add functionality to it). Since it implements an interface that now supports a list and I have no intention of adding an implementation to support it - I do this check.

Reply:
As for checking `Iterable` - well, unfortunately it won't work since a python string is also iterable, so this will cause the command to always fail. I'm keeping the `list` check.